docs/source/elements.ipynb

```diff
@@ -0,0 +1,170 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.imshow(w9_image[1])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def test_beam_statistics_report_only(RE, db, shadow_tes_simulation):\n",
```

Comment:
Is `test_beam_statistics_report_only` here by intention? It seems like it might be left over and should be removed.

Reply:
Good point, was about to suggest @thomaswmorris correct the name of the function.

Reply:
Oops, sphinx was generating from a deprecated version of the notebook. ``make clear`` fixed the issue.